source/cpu/cpu.go

```diff
@@ -197,6 +198,11 @@ func (s *cpuSource) GetLabels() (source.FeatureLabels, error) {
 		labels["hardware_multithreading"] = v
 	}
 
+	// CPU Socket Count
+	if v, ok := features.Attributes[TopologyFeature].Elements["socket_count"]; ok {
+		labels["socket_count"] = v
+	}
```

Comment:
I think we could start with having this only as a raw feature, no label generated by default (https://github.com/kubernetes-sigs/node-feature-discovery/issues/1383#issuecomment-1737335857).

```suggestion
```